spec/system/consumer/shopping/checkout_stripe_spec.rb

```diff
@@ -180,6 +180,8 @@
         end
 
         it "allows saving a card and re-using it" do
+          # TODO: This case fails in production after the activemerchant upgrade.
+          # But this test case still passed. We need to make this spec fail correctly.
```

Comment:
Continuing :duck: 
Agree this one is relevant as well - here we're indeed dealing with :js (browser). So `puffing-billy` would be the way to go to intercept these requests.

I'm thinking though if https://github.com/phillbaker/capybara-mechanize would be suitable here too, in the case we would skip js too :thinking:

Reply:
I don't see how capybara-mechanize would help here. That seems to be a tool to make actual web requests from specs but we want to deal with the requests sent from our app and record real responses to replay.